src/Android/Renderers/CustomEntryRenderer.cs

```diff
@@ -29,7 +29,16 @@ protected override void OnElementChanged(ElementChangedEventArgs<Entry> e)
                     Control.PaddingBottom + 20);
                 Control.ImeOptions = Control.ImeOptions | (ImeAction)ImeFlags.NoPersonalizedLearning |
                     (ImeAction)ImeFlags.NoExtractUi;
-            }
+            }   
+        }
+
+        // Workaround for bug preventing long-press -> copy/paste on Android 11
+        // See https://issuetracker.google.com/issues/37095917
```

Comment:
Wow, I can't believe this has been an issue for 5 years and hasn't been resolved. Seems like a hacky workaround, but it seems like the most accepted workaround